packaging/standalone/src/main/distribution/shell-scripts/bin/Neo4j-Management/Get-Java.ps1

```diff
@@ -146,47 +151,28 @@ Function Get-Java
     if ($javaPath -eq '') { Write-Error "Unable to determine the path to java.exe"; return $null }
     if ($javaCMD -eq '') { $javaCMD = "$javaPath\bin\java.exe" }
     if (-not (Test-Path -Path $javaCMD)) { Write-Error "Could not find java at $javaCMD"; return $null }
-  
+ 
     $ShellArgs = @()
 
+    Write-Verbose "Java detected at '$javaCMD'"
+    Write-Verbose "Java version detected as '$javaVersion'"
```

Comment:
You can, I haven't invested any time into doing it though.  Did you want to squeeze it in for 3.0 if possible or just bump it to 3.1?